doc/tutorials/open_vswitch.rst

```diff
@@ -0,0 +1,160 @@
+.. _open_vswitch:
+
+Using OVS bridge instead of ACRN bridge
+#######################################
+Hypervisors need the ability to bridge network traffic between VMs
+and with the outside world. This tutorial describes how to
+use `Open Virtual Switch (OVS)
+<https://www.openvswitch.org/>`_ bridge instead of ACRN
+bridge for this purpose.
+
+.. note::
+   OVS is provided as part of the ``service-os``
+   bundle.  Use ClearLinux OS version ``28900`` or higher.
+
+What is OVS
+***********
+Open vSwitch (OVS) is an open-source implementation of
+a distributed virtual multilayer switch that provides a switching
+stack for hardware virtualization environments. OVS supports multiple
+protocols and standards used in computer networks, and
+supports the following features:
+
+- Visibility into inter-VM communication via NetFlow,
+  sFlow(R), IPFIX, SPAN, RSPAN, and GRE-tunneled mirrors
+- LACP (IEEE 802.1AX-2008)
+- Standard 802.1Q VLAN model with trunking
+- Multicast snooping
+- IETF Auto-Attach SPBM and rudimentary required LLDP support
+- BFD and 802.1ag link monitoring
+- STP (IEEE 802.1D-1998) and RSTP (IEEE 802.1D-2004)
+- Fine-grained QoS control
+- Support for HFSC qdisc
+- Per VM interface traffic policing
+- NIC bonding with source-MAC load balancing, active backup, and L4 hashing
+- OpenFlow protocol support (including many extensions for virtualization)
+- IPv6 support
+- Multiple tunneling protocols (GRE, VXLAN, STT, and Geneve, with IPsec support)
+- Remote configuration protocol with C and Python bindings
+- Kernel and user-space forwarding engine options
+- Multi-table forwarding pipeline with flow-caching engine
+- Forwarding layer abstraction to ease porting to new software and hardware platforms
+
+For more detailed OVS information, please refer to `<https://www.openvswitch.org/>`_
+
+.. _enable_ovs_in_ACRN:
+
+How to enable OVS in ACRN
+*************************
+The OVS service is installed with the ``service-os`` bundle.
+After booting the ACRN Service OS, start the OVS service with the command::
+
+   # systemctl start openvswitch
+
+To start OVS automatically after a reboot, you should also use this command::
+
+   # systemctl enable openvswitch
+
+.. note::
```

Comment:
This note should be moved to the beginning of this section, and not be a "note":

```
How to enable OVS in ACRN
*************************
The OVS service is included with the Clear Linux ``service-os`` bundle.

After booting the ACRN Service OS, disable the Clear Linux
autoupdate feature before setting up the OVS bridge to
prevent autoupdate from restoring the default bridge after
a system update::

   # swupd autoupdate --disable

You can then start the OVS service with the command::

   # systemctl start openvswitch

To start OVS automatically after a reboot, you should also use this command::

   # systemctl enable openvswitch
```